docs/application/dotnet/guides/nui/nui-components/Pagination.md

```diff
@@ -4,135 +4,222 @@ Pagination is a common component supporting multiple pages. It shows the number
+   and
+   [Class DirectoryInfo](https://docs.tizen.org/application/dotnet/api/tizenfx/api/Tizen.Applications.DirectoryInfo.html)).
+
+   Instead of images one can set a solid color of the indicators: 
+   ```cs
+   _pagination.IndicatorColor = new Color(1.0f, 1.0f, 1.0f, 0.5f);
+   _pagination.SelectedIndicatorColor = Color.Black;
+   ```
+
+3. Add the Pagination to the View:
+
+   ```cs
+   _rootView.Add(_pagination);
+   ```
+
 
 Following output is generated when the Pagination is created using property:
 
-![Pagination1](./media/Pagination1.png)
+| Indicators with images                               | Solid color indicators                               |
+|------------------------------------------------------|------------------------------------------------------|
+|![Pagination1a](./media/PaginationExample_Images.png) | ![Pagination1b](./media/PaginationExample_Color.png) |
+
+
 
+<a name="createWithStyle"></a>
 ## Create with style
 
 To create a Pagination using style, follow these steps:
 
 1. Create a style for Pagination:
 
     ```cs
-    PaginationStyle style = new PaginationStyle()
+   string _imageUrl = Tizen.Applications.Application.Current.DirectoryInfo.Resource + "images/";
+
+    PaginationStyle _style = new PaginationStyle()
     {
-        IndicatorSize = new Size(15, 15),
-        IndicatorSpacing = 20,
+        IndicatorSize = new Size(100, 100),
+        IndicatorSpacing = 30,
         IndicatorImageUrl = new Selector<string>
         {
-            Normal = "pagination_ic_nor.png",
-            Selected = "pagination_ic_sel.png"
+            Normal = _imageUrl + "shots.jpg",
+            Selected = _imageUrl + "shots.gif"
         },
         Name = "Pagination2",
-        Size = new Size(300, 50),
-        BackgroundColor = new Color(1.0f, 1.0f, 1.0f, 0.6f),
+        Size = new Size(600, 200),
+        BackgroundColor = new Color(0.0f, 0.0f, 0.0f, 1.0f)
     };
     ```
 
-2. Use the style to create a Pagination and add it to parent:
+2. Use the style to create a Pagination (regardless of the previous example):
 
     ```cs
-    pagination = new Pagination(style);
-    pagination.IndicatorCount = 5;
-    pagination.SelectedIndex = 2;
-    root.Add(pagination);
+    var _pagination = new Pagination(_style);
+    _pagination.IndicatorCount = 3;
+    _pagination.SelectedIndex = 2;
     ```
 
+3. Add the Pagination to the parent:
+
+   ```cs
+   _rootView.Add(_pagination);
+   ```
+
+
 Following output is generated when the Pagination is created using style:
 
-![Pagination2](./media/Pagination2.png)
+![Pagination2](./media/PaginationExample_Animated.gif)
+
+
 
+<a name="createWithCustomStyle"></a>
 ## Create with defined styles
 
 You can define a style based on the user experience (UX) and then use this style to create a Pagination.
 
-1. Define a custom style:
+1. Define a custom style inside you namespace (called `YourNameSpace` in this example):
```

Comment:
```suggestion
1. Define a custom style inside the namespace (called `YourNameSpace` in the following example): 
```

Where are we using `YourNameSpace` ?? Please verify.

Reply:
in the second point of this example - I added this info to the text

Reply:
changed as suggested by mijin85cho